v4/export/writer_util.go

```diff
@@ -108,37 +86,127 @@ func wrapStringWith(str string, wrapper string) string {
 	return fmt.Sprintf("%s%s%s", wrapper, str, wrapper)
 }
 
-func convert(origin []sql.NullString, colTypes []string) []string {
-	ret := make([]string, len(origin))
-	for i, s := range origin {
-		if !s.Valid {
-			ret[i] = "NULL"
-			continue
+func SQLTypeStringMaker() RowReceiverStringer {
+	return &SQLTypeString{}
+}
+
+func SQLTypeBytesMaker() RowReceiverStringer {
+	return &SQLTypeBytes{}
+}
+
+func SQLTypeIntegerMaker() RowReceiverStringer {
+	return &SQLTypeInteger{}
+}
+
+var colTypeRowReceiverMap = map[string]func() RowReceiverStringer{
+	"CHAR":      SQLTypeStringMaker,
+	"NCHAR":     SQLTypeStringMaker,
+	"VARCHAR":   SQLTypeStringMaker,
+	"NVARCHAR":  SQLTypeStringMaker,
+	"TEXT":      SQLTypeStringMaker,
+	"ENUM":      SQLTypeStringMaker,
+	"SET":       SQLTypeStringMaker,
+	"JSON":      SQLTypeStringMaker,
+	"TIMESTAMP": SQLTypeStringMaker,
+	"DATETIME":  SQLTypeStringMaker,
+	"DATE":      SQLTypeStringMaker,
+	"TIME":      SQLTypeStringMaker,
+	"YEAR":      SQLTypeStringMaker,
+
+	"BINARY":    SQLTypeBytesMaker,
+	"VARBINARY": SQLTypeBytesMaker,
+	"BLOB":      SQLTypeBytesMaker,
+	"BIT":       SQLTypeBytesMaker,
+
+	"INT":     SQLTypeIntegerMaker,
```

Comment:
Can this handle MEDIUMBLOB / BIGINT / etc?

Reply:
Sure, we can add fill this map gradually.

Reply:
Ok.